lib/FrontendTool/FrontendTool.cpp

```diff
@@ -798,8 +798,7 @@ static bool performCompile(CompilerInstance &Instance,
   if (auto SM = Instance.takeSILModule()) {
     PSGIs.push_back(PostSILGenInputs{std::move(SM), false, mod});
   }
-
-  if (PSGIs.empty()) {
+  else if (PSGIs.empty()) {
```

Comment:
This is just `else`. The deque has just been declared.

Reply:
Ah--done!